tools/lsp/ui/views/property-view.slint

```diff
@@ -112,18 +112,19 @@ export component PropertyView {
                                             checked: property-row.property-value.value_bool;
 
                                             toggled() => {
-                                                Api.set-bool-binding(
+                                                Api.set-code-binding(
                                                     root.current-element.source-uri,
                                                     root.current-element.source-version,
                                                     root.current-element.range.start,
                                                     property.name,
-                                                    self.checked,
+                                                    self.checked ? "true" : "false",
                                                 );
                                             }
                                         }
                                         if property-row.property-value.kind == PropertyValueKind.string: LineEdit {
                                             width: 100%;
-                                            height: 100%; // otherwise this gets too high and covers several rows.
+                                            height: 100%;
+                                            // otherwise this gets too high and covers several rows.
```

Comment:
You should either not use the broken formater, or not have comments after in the same line.  (more occurences)